frontend/apps/marketing/src/config/jsonLd/OrganizationJsonLd.tsx

```diff
@@ -0,0 +1,77 @@
+import {JsonLd} from 'react-schemaorg';
+import {WithContext, EducationalOrganization} from 'schema-dts';
+
+import {Brand} from '@/config/brand';
+
+/** See: https://developers.google.com/search/docs/appearance/structured-data/organization */
+const BRANDS_ORG_JSON_LD: {
+  [K in Brand]?: WithContext<EducationalOrganization>;
+} & {
+  [Brand.CODE_DOT_ORG]: WithContext<EducationalOrganization>;
```

Comment:
```suggestion
  'code.org': WithContext<EducationalOrganization>;
```

This should fix the unit test failure